module/People/src/People/Controller/UserProfileController.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace People\Controller;
+
+use Application\Entity\User;
+use People\Service\OrganizationService;
+use Application\Controller\OrganizationAwareController;
+use Application\Service\UserService;
+use People\View\UserProfileJsonModel;
+use Accounting\Service\AccountService;
+use Accounting\Entity\PersonalAccount;
+
+class UserProfileController extends OrganizationAwareController
+{
+	protected static $collectionOptions = array('GET');
+	protected static $resourceOptions = array('DELETE', 'POST', 'GET', 'PUT');
+	
+	private $orgService;
+	private $userService;
+	private $accountService;
+	
+	public function __construct(OrganizationService $orgService, UserService $userService, AccountService $accountService) {
+		$this->orgService = $orgService;
+		$this->userService = $userService;
+		$this->accountService = $accountService;
+	}
+	
+	public function get($id)
+	{		
+		if(is_null($this->identity())) {
+			$this->response->setStatusCode(401);
+			return $this->response;
+		}
+		
+		if(!$this->isAllowed($this->identity(), $this->organization, 'People.UserProfile.getProfile')) {
```

Comment:
La `isAllowed` deve controllare che l'utente corrente possa accedere al profilo dell'utente richiesto, quindi che `$this->isAllowed($this->identity(), $user, 'People.UserProfile.profile?') in cui la assertion controlla che sia l'uno che l'altro appartengano all'organizzazione in cui si sta lavorando. In questo momento potresti avere difficoltà a capire nell'Assertion qual è l'organizzazione in cui si sta lavorando. Se così fosse, per il momento, va bene verificare che i due utenti appartengano almeno ad un'organizzazione in comune